packages/core/helper-plugin/lib/src/components/NoContent/NoContent.stories.mdx

```diff
@@ -0,0 +1,25 @@
+<!--- EmptyStateLayout.stories.mdx --->
+
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { Main, Row, Button } from '@strapi/parts';
+import NoContent from './index';
+
+<Meta title="components/NoContent" />
+
+# NoContent
+
+This component is used to display an empty state.
+
+## Usage
+
+<Canvas>
+  <Story name="base">
+    <Main>
+      <NoContent content="There's no content" action={<Button>Add content</Button>} />
```

Comment:
Can you add the content as a defaultProps so we don't have to recreate the translation nor look up for one each time we use this component?